### DISCUSSION TITLE
Ring app status mismatch for certain devices?

### DISCUSSION THREAD

**Discussion by ajfi-droid2** [tsightler/ring-mqtt#810] [Support Q&A] [answered]
First off, apologies if I’m asking a question answered elsewhere - I did check the FAQs and skim the documentation, but I admit that this may still be a dumb question. 
I noticed that when I take certain actions in Home Assistant (eg toggling my floodlight camera’s lights), the Ring app does not reflect the updated status of the system (in the example given, it doesn’t show the floodlight camera’s lights as on). 
I was just wondering if this is a known limitation of Ring-MQTT, or an issue with my installation? 
Thanks to all who read this, and thanks especially to the creator of this incredible add-on!

**Comment by tsightler** [comment, marked as answer]
This is not a limitation of ring-mqtt, it is a limitation of the Ring API.  Ring cameras use a polling interface which is not real-time.  In many cases the state is not properly reflected in the Ring API for 30 seconds or more.  Even with two phones with the Ring app installed you can see this, turn on the light on one phone, and refresh on the other phone, it will almost always show the light as off.  But sometime around 20-30 seconds a refresh will finally show the correct state.

Ring apps actually don't poll at all, they wait for you to manually refresh the state, but this doesn't work for a project designed for automation.  The ring-mqtt app polls every 20 seconds, but realistically, the API updates even less frequently than that, and polling more often just leads to you being throttled by Ring servers more quickly, not in faster response.

**Reply by ajfi-droid2** [reply]
Thanks for responding and explaining!
